instrumentation/http/src/main/java/brave/http/HttpServerHandler.java

```diff
@@ -148,22 +151,12 @@ Span nextSpan(TraceContextOrSamplingFlags extracted, HttpServerRequest request)
    * @since 4.3
    */
   public void handleSend(@Nullable Resp response, @Nullable Throwable error, Span span) {
-    if (response == null && error == null) {
```

Comment:
Keep this check?

Reply:
same answer as other copy/paste. we defer the check to the super